integration_test/install_test.go

```diff
@@ -141,6 +141,18 @@ func TestKrewInstall_CustomIndex(t *testing.T) {
 	test.AssertExecutableNotInPATH("kubectl-" + validPlugin2)
 }
 
+func TestKrewInstall_CustomDefaultIndexURI(t *testing.T) {
```

Comment:
I don't think this test is necessary. We don't need to check all our machinery with & without setting all env variables. You can't do that exhaustively anyway, because you get a factorial explosion of test cases.

I'd recommend to have a single simple test that checks if the env var gets picked up.